### DISCUSSION TITLE
非常棒的作品，希望主动打赏

### DISCUSSION THREAD

**Discussion by Big1st** [xiaye13579/BBLL#546] [General]
Dear xiaye13579，
        我是一名很喜欢用电视观看bilibili的用户，每天睡前躺着的时候，观看b站就是我最好的放松方式。之前B站TV版还是挺好用的，我记得应该是2022年上半年的时候。
        后面更新为“云视听小电视”以后，我没办法在TV版里面关注我喜欢的up主，也没办法第一时间找到自己已关注的up主的动态，其实对用户来说带去了非常多的不便，我相信其他用户也有相同的感受。我相信这也是您的作品这么受欢迎的原因。
        无论您的作品是否存在bug，您一直都在积极维护，为喜爱它的用户不断地完善体验，数以万计的人从您的作品中受益，包括我，所以我个人是非常感激的。
        希望在保持软件免费的前提下，可以开放渠道对您的作品进行打赏，这个行为完全是我自发的，希望在表达感激之情的的同时，可以更好支持您为了该项目的付出。
        谢谢！

**Comment by JSSBach** [comment]
是的，我也愿意资助作者。感谢！

**Comment by jadexb** [comment]
me too

**Comment by aihuijin** [comment]
支持！！！

**Comment by wadn2008** [comment]
支持，同求捐赠渠道

**Comment by MasonWest** [comment]
支持支持，非常不喜欢云试听小电视

**Comment by tingkau** [comment]
支持
